tests/all/js_globals/JsString.rs

```diff
@@ -261,6 +261,45 @@ fn index_of() {
         .test()
 }
 
+#[test]
+fn last_index_of() {
+    project()
+        .file("src/lib.rs", r#"
+            #![feature(proc_macro, wasm_custom_section)]
```

Comment:
I believe to pass tests this'll need to change `proc_macro` to `use_extern_macros`

Reply:
Ah! Sure thing, thanks for the review.